cdc/kv/client.go

```diff
@@ -689,13 +689,13 @@ func (s *eventFeedSession) requestRegionToStore(
 		state := newRegionFeedState(sri, requestID)
 		pendingRegions.setByRequestID(requestID, state)
 
-		log.Debug("start new request",
+		log.Info("start new request",
```

Comment:
Is this necessary to help debug? Will this cause too much verbose logs ?

Reply:
It's very helpful, we can trace lifetime of all regions with the log.